sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/HDFSMetadataLog.scala

```diff
@@ -46,8 +46,7 @@ import org.apache.spark.sql.SQLContext
  * files in a directory always shows the latest files.
  */
 class HDFSMetadataLog[T: ClassTag](sqlContext: SQLContext, path: String)
-  extends MetadataLog[T]
-  with Logging {
+  extends MetadataLog[T] with Logging {
```

Comment:
Just not worth changing on its own. I am not sure there's a strong style preference here. It's not a typo and that seems to be the logical purpose of this PR.